src/app/content/reducer.ts

```diff
@@ -88,7 +88,10 @@ function reduceContent(state: State, action: AnyAction) {
       return reduceReceiveBook(state, action);
     }
     case getType(actions.requestPage):
-      return {...state, loading: {...state.loading, page: action.payload}};
+      return {
+      ...(isEqual(action.payload, state.page) ? state : omit(['page'], state)),
```

Comment:
the condition you have here will always be false, i think maybe you wanted to do 
```suggestion
      ...(isEqual(action.payload, state.params.page) ? state : omit(['page'], state)),
```
but i'm realizing that won't work either, because on a location change, the state.params will have just been updated to be the new ones.

i apologize apparently the logic of how all this works has fallen out of my head, digging back around again i see that the case of navigating to another location on the same page is short circuited [here](https://github.com/openstax/rex-web/blob/main/src/app/content/hooks/locationChange/resolveContent.ts#L159), and in normal rex behavior `requestPage` is only ever called when we _really_ want to request the page. in the assignable layout we also know for sure that we want to request the page

so long story short i think you were right the first time, you can remove the extra conditional logic. let me know if this is making sense to you, its possible i'm still confused...

Reply:
ugh actually what I meant to do was check the page params against each other - something like `isEqual(action.payload.uuid, state.page.id)` but I just realized the payload could be of type SlugParams. also when loadPage is called from resolveContent, there is already a check to ensure a different page is being loaded. so I think you're right that the conditional is not necessary.